### ISSUE TITLE
Cy.log to print to the UI left column

### ISSUE THREAD

**Issue by bahmutov** [cypress-io/cypress#342]
Sometimes it is hard to understand which part of the test is executing, especially if there are a lot of commands. It would be nice to include simple text messages in the left output column of the UI
Something like this

```js
cy.get(...)
cy.log('testing login')
cy.get(...)
```

Currently I print to the dev tools console using
```js
cy.then(() => console.log('something something')
```

**Comment by bahmutov** [comment]
Here is your implementation
```js
// a useful command to log message with arguments to the left
// UI panel.
Cypress.addParentCommand('log', function (msg, args) {
  Cypress.Log.command({
    name: 'log',
    message: [msg, args],
    end: true,
    snapshot: true,
    consoleProps: function () {
      return {
        message: msg,
        args: args
      }
    }
  })
  return null
})
```

and the left panel screenshot
<img width="382" alt="cy-log" src="https://cloud.githubusercontent.com/assets/2212006/21152548/b342f78e-c134-11e6-8e94-bcf036fa2f61.png">

**Comment by brian-mann** [comment]
Fixed in [`0.18.3`](https://github.com/cypress-io/cypress/wiki/Changelog#0183-12182016).
